src/liballoc/heap.rs

```diff
@@ -27,36 +27,46 @@ pub mod __core {
 
 extern "Rust" {
     #[allocator]
+    #[allocator_nounwind]
```

Comment:
Hmm, this is a new ungated attribute, right?
Since it's internal and the name doesn't matter much, `rustc_allocator_nounwind` can be used instead, it'll be gated automatically.